### DISCUSSION TITLE
Manual Disable Decky

### DISCUSSION THREAD

**Discussion by jwagman1** [SteamDeckHomebrew/decky-loader#351] [Q&A] [answered]
Every time I start SteamOS it crashes at the user login screen. I am pretty sure one of the Decky plug-ins is causing the crash. so I can't boot into desktop mode to access the console/uninstall options.

Is there any way to delete a file or directory that will disable decky so I can boot into SteamOS and fix it? I have Windows 11 dual booted with the SteamOS partitions mounted. I can't find the files tho.

**Comment by PartyWumpus** [comment, marked as answer]
`/home/deck/homebrew/` is the main folder
To disable decky you can just delete the `/home/deck/homebrew/services/PluginLoader` file
If you want to delete your plugins you can delete the `/home/deck/homebrew/plugins/` folder

**Reply by TrainDoctor** [reply]
Just to add onto this, if you're able to establish file access to the deck this is the best option. That being deleting the PluginLoader binary. If you can get SSH access to the deck the better option is to disable ``plugin_loader.service``. I would also note that we don't provide support for dual boot configurations as windows has a tendency to break anything that isn't to it's liking.
